app/models.py

```diff
@@ -95,19 +95,28 @@ def get_url(self, expires_in=3600):
         return s3.get_url(self.path, expires_in)
 
 
-class PlanEnum(enum.Enum):
+class EnumE(enum.Enum):
+    @classmethod
+    def has_value(cls, value: int) -> bool:
+        return value in cls._value2member_map_
```

Comment:
I wonder if we can use this code that doesn't use the private attribute `_value2member_map_`:

```
def has_value(cls, value: int) -> bool:
     try:
           cls(value)
     except ValueError:
           return False
     else:
           return True
```